iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/connector/client/IoTDBDataNodeAsyncClientManager.java

```diff
@@ -153,7 +156,7 @@ public AsyncPipeDataTransferServiceClient borrowClient(TEndPoint endPoint) throw
       if (handshakeIfNecessary(endPoint, client)) {
         return client;
       }
-    } catch (Exception e) {
+    } catch (final Exception e) {
```

Comment:
hi , I want to learn some thing,  why is final exception ?

Reply:
Just a style of code